### PR TITLE
Add Aikatsu Stars! My Special Appeal

### PR DESCRIPTION
**Pull request by denliner**
Adds Aikatsu Stars! My Special Appeal.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/aikatsu-stars-my-special-appeal/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Aikatsu Stars! My Special Appeal"
+description = "Aikatsu Stars! My Special Appeal  is a 3D rhythm game developed and published by Bandai Namco Games"
```
Comment (CodingKoopa) [inline games/aikatsu-stars-my-special-appeal/game.dat]: Remove one of the spaces after "Appeal".

Comment (CodingKoopa) [inline games/aikatsu-stars-my-special-appeal/game.dat]: Add a period at the end.

Comment (Leo626) [inline games/aikatsu-stars-my-special-appeal/game.dat]: Also you could add these sentences. Should mention that there is a free-to-play version.

`The game is the fifth entry in the Aikatsu series. It had a free-to-play version called \"Aikatsu Stars! First Appeal\".`

**Comment by Leo626** [comment]
Still need to create the wiki page. See the [contributing guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#wiki) for example.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

**Pull request closed without merging**